lib/shopify_app/configuration.rb

```diff
@@ -15,6 +15,9 @@ class Configuration
     attr_accessor :scripttags
     attr_accessor :after_authenticate_job
     attr_accessor :session_repository
+    attr_accessor :per_user_tokens
+    alias_method :per_user_tokens?, :per_user_tokens
+    attr_accessor :scope
```

Comment:
@ashchristopher what was this added for?

Reply:
It can be removed - I think it may be due to fixing merge conflicts. It is defined on line 11, so let's get rid of it (assuming you are referring to `attr_accessor :scope`)